hack/metadata.schema.json

```diff
@@ -103,6 +103,14 @@
           }
         }
       }
+    },
+    "clusterVersions": {
+      "type": "array",
+      "description": "The cluster versions required to execute the script",
+      "additionalProperties": false,
+      "items": {
+        "type": "string"
```

Comment:
Also, do you think we should add a pattern field under the type like:

Reply:
I think it's just a nit which is relevant to this ticket, and I posted the comment 2 weeks ago. Any reason you want to have another ticket for that or any concerns or questions about the pattern? 

I don't have approval permission, if you want to have the current code merged and closeed in this sprint, please ask Bo to approve and I am all good.